### DISCUSSION TITLE
Optimized settings for heavy simulations

### DISCUSSION THREAD

**Discussion by Perpetual3134** [InteractiveComputerGraphics/SPlisHSPlasH#278] [General]
Hi there,

I wanted to ask you what settings you adjust to calculate large simulations with a lot of particles? 
(e.g. about 40 000 000 particles)

**Comment by janbender** [comment]
The physical parameters are the same for large scale simulations. Make sure that if you use very small particles that you also increase the resolution of the volume maps (if you use this boundary handling).
